tests/spec/examples.py

```diff
@@ -796,6 +777,50 @@ def it_runs_before_starting_a_service(self):
         )
 
 
+class DescribePostStartHook(object):
+
+    @pytest.yield_fixture
+    def service_name(self):
+        yield 'post-start-hook'
+
+    @pytest.mark.usefixtures('in_example_dir')
+    def it_runs_after_all_services_have_started(self):
+        assert_command(
+            ('pgctl', 'start', 'A'),
+            '',
+            '''\
+[pgctl] Starting: A
+[pgctl] Started: A
+''',
+            0,
+            norm=norm.pgctl,
+        )
+        assert_command(
+            ('pgctl', 'start', 'B'),
+            '',
+            '''\
+[pgctl] Starting: B
+[pgctl] Started: B
+hello, i am a post-start script
```

Comment:
I believe we can run `pgctl status` in playground-wide pre-start, post-start, and post-stop hooks because these hooks are run outside of a lock.

Also, I agree that triggering these playground-wide hooks upon every `pgctl start/stop xxx` is a more flexible design that allows a myriad of possibilities being implemented in the hooks. Of course, the downside is that users would have to take care of the conditioning based on playground services state in the hooks.

I have no strong leaning toward either but I would love to see the three playground-wide hooks behave consistently.